SimPEG/EM/Static/DC/IODC.py

```diff
@@ -0,0 +1,444 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib
+import properties
+import warnings
+
+import SimPEG
+from SimPEG import Utils, Mesh
+from . import SrcDC as Src   # Pole
+from . import RxDC as Rx
+from .SurveyDC import Survey_ky
+
+
+class IO(properties.HasProperties):
+    """ """
+
+    # Survey
+    survey_geometry = properties.StringChoice(
```

Comment:
I find `survey_geometry` and `survey_type` potentially confusing (`geometry` suggests survey layout, and might suggest to the user that it is `pole-pole` etc). 

The `survey_geometry` is maybe more `survey_layout`? thoughts @sgkang, @thast

Reply:
good point. `survey_layout` is fine for me.

Reply:
I like layout better too for this case